loader/src/main/java/net/neoforged/fml/loading/FMLConfig.java

```diff
@@ -81,12 +86,14 @@ private static Object maxThreads(final Object value) {
 
     private static final Logger LOGGER = LogUtils.getLogger();
     private static final FMLConfig INSTANCE = new FMLConfig();
+    private static final Map<String, List<DependencyOverride>> DEPENDENCY_OVERRIDES = new HashMap<>();
```

Comment:
Nitpick: this is a mutable field so it should use lower camel case, even if it's constant. :P

Reply:
The field is not mutable, it's final.

Reply:
It is final, but it contains a mutable value. Just look at the lines below :P